app/frontend/src/service/client.ts

```diff
@@ -6,7 +6,7 @@ import { JailPromiseClient } from "../pb/jail_grpc_web_pb";
 import { RequestsPromiseClient } from "../pb/requests_grpc_web_pb";
 import { SSOPromiseClient } from "../pb/sso_grpc_web_pb";
 
-const URL = process.env.REACT_APP_API_BASE_URL;
+export const URL = process.env.REACT_APP_API_BASE_URL;
```

Comment:
This name could be more descriptive, especially now it is exported.
```suggestion
export const API_BASE_URL = process.env.REACT_APP_API_BASE_URL;
```

Reply:
But maybe it is better to just use `process.env.REACT_APP_API_BASE_URL` where you need it.